modules/zone/variables.tf

```
@@ -1,10 +1,21 @@
 variable "name" {
   description = "Name of the zone to create"
   type        = string
+  default     = null
 }
 
 variable "tags" {
   description = "Map of tags to apply to the Route53 zone"
   type        = map(string)
   default     = {}
 }
+
+variable "vpc_id" {
```

Suggest modifying the variable name to match the block name where it is used...

```suggestion
variable "vpc" {
```